### ISSUE TITLE
Organization of Section 3

### ISSUE THREAD

**Issue by rartino** [Materials-Consortia/OPTiMaDe#155]
Moved to a separate issue from #141:

"Section 3.3.4.  Properties with unknown value" is not about responses, but more about a general feature of OPTiMaDe and filtering. This should NOT be a subsection to 3.3 Responses, but rather directly under section 3.

I also think some re-ordering of the subsections of section 3 would likely improve organization.

However, I propose to do this only after #103 is done.

**Comment by rartino** [comment]
Resolved by #189
